jenkins-design-language/src/js/components/forms/Dropdown.jsx

```diff
@@ -265,6 +265,11 @@ export class Dropdown extends React.Component {
         if (this.props.onChange) {
             this.props.onChange(option, index);
         }
+
+        //restore the focus on the button element in IE
+        if (this.buttonRef.setActive) {
```

Comment:
Should we have check if the browser in context is IE or I guess there is no side effect for other browsers?

Reply:
setActive is an IE only method, that is why I checked for its existence before calling it, this ensures it runs only in IE

Reply:
@NicuPascu I just tested with firefox and chrome and the loose the focus in showing there as well. Trying to say I think you need to change the above to 

```
if (this.buttonRef && this.buttonRef.focus) {
  this.buttonRef.focus();
}
```

This will further prevent possible NPE like `undefined.focus`